### PR TITLE
Fixes #15660: Print command output during upgrade steps in debug color

### PR DESCRIPTION
**Pull request by ehelms**
This is especially useful for Pulp migrations as they contain useful
information regarding what stage they are at and if they will be long
running.

### CODE REVIEW

File: hooks/boot/01-helpers.rb [hunk 1]
```diff
@@ -8,6 +8,7 @@ def module_enabled?(kafo, name)
 
     def log_and_say(level, message)
       style = level == :error ? 'bad' : level
+      style = :yellow if level == :debug
```
Comment (stbenjam) [inline hooks/boot/01-helpers.rb]: Do you think a case statement is clearer? Something like

```ruby
style = case level
        when :error
          'bad'
        when :debug
          'yellow'
        else
          level
        end
```

**Comment by stbenjam** [comment]
But otherwise :+1: :+1:

**Comment by ehelms** [comment]
@stbenjam updated with the case statement

**Comment by stbenjam** [comment]
ACK

**Pull request merged**